public/scss/_events.scss

```diff
@@ -64,6 +74,16 @@
     border: 1px solid $dark-grey;
 }
 
+.event-attendees-table {
+    background-color: #ffffff;
+}
+
+.email-column-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
```

Comment:
delete all that because it causes a weird double border, instead put `float: right` on the button itself